src/shared/context/TableContext/tableReducer.ts

```diff
@@ -1,8 +1,7 @@
 import { TableAction, TableState } from './types'
```

Comment:
I had to update the table type system a bit to make it easier to work with (blame TS).